benchmarks/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Adding this feels way more cumbersome than just adding `benchmark` as a devDependency.